tests/sentry/deletions/test_group.py

```diff
@@ -13,32 +13,46 @@
     ScheduledDeletion,
     UserReport,
 )
+from sentry import nodestore
+from sentry.deletions.defaults.group import GroupNodeDeletionTask
 from sentry.tasks.deletion import run_deletion
-from sentry.testutils import TestCase
+
+from sentry.testutils import TestCase, SnubaTestCase
 from sentry.testutils.helpers.datetime import iso_format, before_now
 
 
-class DeleteGroupTest(TestCase):
+class DeleteGroupTest(TestCase, SnubaTestCase):
     def test_simple(self):
-        key = "key"
-        value = "value"
-
+        GroupNodeDeletionTask.DEFAULT_CHUNK_SIZE = 1  # test chunking logic
         event_id = "a" * 32
+        event_id2 = "b" * 32
         project = self.create_project()
+        node_id = Event.generate_node_id(project.id, event_id)
+        node_id2 = Event.generate_node_id(project.id, event_id2)
+
         event = self.store_event(
             data={
                 "event_id": event_id,
-                "tags": {key: value},
+                "tags": {"foo": "bar"},
+                "timestamp": iso_format(before_now(minutes=1)),
+                "fingerprint": ["group1"],
+            },
+            project_id=project.id,
+        )
+
+        self.store_event(
+            data={
+                "event_id": event_id2,
                 "timestamp": iso_format(before_now(minutes=1)),
+                "fingerprint": ["group1"],
             },
             project_id=project.id,
         )
+
```

Comment:
I'd advice you added a third event with a different fingerprint and assert that event is not removed from nodestore.